src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3932,74 +3942,84 @@ public static SubnetGroups getInternodeErrorReportingExclusions()
         return conf.internode_error_reporting_exclusions;
     }
 
-    public static boolean getTrackWarningsEnabled()
+    public static boolean getReadThresholdsEnabled()
     {
-        return conf.track_warnings.enabled;
+        return conf.read_thresholds_enabled;
     }
 
-    public static void setTrackWarningsEnabled(boolean value)
+    public static void setReadThresholdsEnabled(boolean value)
     {
-        conf.track_warnings.enabled = value;
+        if (conf.read_thresholds_enabled != value)
+        {
+            conf.read_thresholds_enabled = value;
+            logger.info("updated read_thresholds_enabled to {}", value);
+        }
     }
 
-    public static long getCoordinatorReadSizeWarnThresholdKB()
+    public static DataStorageSpec getCoordinatorReadSizeWarnThreshold()
     {
-        return conf.track_warnings.coordinator_read_size.getWarnThresholdKb();
+        return conf.coordinator_read_size_warn_threshold;
     }
 
-    public static void setCoordinatorReadSizeWarnThresholdKB(long threshold)
+    public static void setCoordinatorReadSizeWarnThreshold(DataStorageSpec value)
     {
-        conf.track_warnings.coordinator_read_size.setWarnThresholdKb(threshold);
+        logger.info("updated coordinator_read_size_warn_threshold to {}", value);
```

Comment:
Nit: if this is before actually setting the property the message should be `updating coordinator_read_size_warn_threshold to {}`. Or it could be after setting the property if we want to use `updated`. The same applies to the other two properties.